docs/plugins/feature-flags.md

```diff
@@ -1,16 +1,18 @@
 ---
 id: feature-flags
 title: Feature Flags
-description: Details the process of defining setting and reading a plugin feature flag.
+description: Details the process of defining setting and reading a feature flag.
 ---
 
-Backstage offers the ability to define feature flags inside a plugin. This allows you to restrict parts of your plugin to those individual users who have toggled the feature flag to on.
+Backstage offers the ability to define feature flags inside a plugin or application. This allows you to restrict parts of your plugin to those individual users who have toggled the feature flag to on.
 
-This page describes the process of defining setting and reading a plugin feature flag. If you are looking for using feature flags with software templates that can be found under [Writing Templates](https://backstage.io/docs/features/software-templates/writing-templates#remove-sections-or-fields-based-on-feature-flags).
+This page describes the process of defining setting and reading a feature flag. If you are looking for using feature flags with software templates that can be found under [Writing Templates](https://backstage.io/docs/features/software-templates/writing-templates#remove-sections-or-fields-based-on-feature-flags).
 
 ## Defining a Feature Flag
 
-Before using a feature flag we must first define it. This is done when we create the plugin by passing the name of the feature flag into the `featureFlags` array.
+Before using a feature flag we must first define it. This is done when we create the plugin or app by passing the name of the feature flag into the `featureFlags` array.
```

Comment:
Maybe instead of covering both cases into one paragraph, lets try breaking them up. It makes it easier for the reader to see what example belongs to which usecase.

Reply:
True, will change this